driver/modern_bpf/shared_definitions/struct_definitions.h

```diff
@@ -79,3 +79,16 @@ struct counter_map
 	uint64_t n_drops_buffer_proc_exit;
 	uint64_t n_drops_max_event_size; /* Number of drops due to an excessive event size (>64KB). */
 };
+
+struct filter_map_entry
+{
+	uint16_t arg_num;	       /* Arg number containing strings to compare against prefixes */
+	uint16_t num_prefixes;     /* Length of prefix list */
+	uint8_t prefixes[32][32];  /* List of prefixes to filter */
```

Comment:
From PR review: Should this be changed to `[12][32]` to match the ARM limitation?